src/main/kotlin/car_racing/CarListManager.kt

```diff
@@ -0,0 +1,26 @@
+package car_racing
+
+class CarListManager(cars: List<Car> = mutableListOf()) {
+    private val _cars: List<Car> = cars
+    val cars: List<Car>
+        get() = _cars
```

Comment:
여기는 Backing Properties가 필요없어보여요! 한번 검토부탁드려요~!!

```suggestion
class CarListManager(private val cars: List<Car> = listOf()) {
```